src/test/java/org/reaktivity/nukleus/http_cache/internal/streams/proxy/Rfc7234ProxyIT.java

```diff
@@ -655,7 +655,7 @@ public void shouldRevalidateOnResponseNoCache() throws Exception
         k3po.finish();
     }
 
-    @Ignore
+    @Ignore("if the Vary header value is * then request considered as not matched and cache should be invalidated.")
     @Test
```

Comment:
File issue to cover this remaining bug.

Then, refer to the issue number only inside `@Ignore` and put the explanation in the issue description instead.